smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HTTPResponseTraitWithoutHTTPPayload.kt

```diff
@@ -50,40 +51,39 @@ class XMLHttpResponseTraitWithoutHttpPayload(
         val memberName = ctx.symbolProvider.toMemberName(streamingMember.member)
         when (shape.type) {
             ShapeType.UNION -> {
-                writer.openBlock("if case let .stream(stream) = httpResponse.body, let responseDecoder = decoder {", "} else {") {
-                    writer.declareSection(HttpResponseTraitWithHttpPayload.MessageDecoderSectionId) {
-                        write("let messageDecoder: \$D", ClientRuntimeTypes.EventStream.MessageDecoder)
-                    }
+                writer.openBlock("if case let .stream(stream) = httpResponse.body {", "} else {") {
+                    writer.write("let messageDecoder = \$N()", customizations.messageDecoderSymbol)
```

Comment:
Once again, customizations are used for message decoder type instead of a section writer.